submissions/ik-web/html-css-popup/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-CSS-Popup</title>
+    <link rel="stylesheet" href="css/app.css">
+</head>
+<body>
+
+    <!-- Nav container -->
+    <nav class="nav">
+        <ul class="nav-inner">
+            <li><a href="#" class="nav-item nav-link">Mail</a></li>
+            <li><a href="#" class="nav-item nav-link">Images</a></li>
+
+            <!-- Popup menu container -->
+
+            <!-- Nav popup button -->
+            <li>
+                <input type="checkbox" id="popup">
+                <label class="nav-item nav-icon" for="popup" tab><img src="icons/popup-button.png" alt="popup-button"></label>
```

Comment:
Attribute `tab` not allowed on element `label`.

Reply:
I'm sorry ... I cleaned it up badly after trying to use the tabindex commands :(